ui/apps/platform/src/Components/SelectSingle/SelectSingle.tsx

```diff
@@ -3,6 +3,7 @@ import { Select, SelectVariant } from '@patternfly/react-core';
 
 export type SelectSingleProps = {
     toggleIcon?: ReactElement;
+    toggleAriaLabel?: string;
```

Comment:
This was added to get a better selector hook for the element that also adds some better accessibility.